tests/integration-tests/tests/storage/storage_common.py

```diff
@@ -161,18 +161,24 @@ def _write_user_data(efs_id, random_file_name):
         """  # noqa: E501
 
 
-def test_efs_correctly_mounted(remote_command_executor, mount_dir):
-    logging.info("Testing ebs {0} is correctly mounted".format(mount_dir))
-    result = remote_command_executor.run_remote_command("df | grep '{0}'".format(mount_dir))
+def test_efs_correctly_mounted(remote_command_executor, mount_dir, tls=False):
+    logging.info("Checking efs {0} is correctly mounted".format(mount_dir))
+    result = remote_command_executor.run_remote_command("mount | column -t | grep '{0}'".format(mount_dir))
     assert_that(result.stdout).contains(mount_dir)
-
-    result = remote_command_executor.run_remote_command("cat /etc/fstab")
-    assert_that(result.stdout).matches(
-        (
-            r".* {mount_dir} nfs4 nfsvers=4.1,rsize=1048576,wsize=1048576,hard,"
-            r"timeo=30,retrans=2,noresvport,_netdev 0 0"
-        ).format(mount_dir=mount_dir)
+    if tls:
+        logging.info("Checking efs {0} enables in-transit encryption".format(mount_dir))
+        assert_that(result.stdout).contains("127.0.0.1:/")
```

Comment:
I'm not getting why `127.0.0.1:/` would mean that in-transit encryption is enabled?

Reply:
I will add a description in the comment
https://docs.aws.amazon.com/efs/latest/ug/encryption-in-transit.html

Reply:
Yes, let's add a comment in the code for this check